functional_testing/Grid_API_Testing/api_testing/testcases/extend_tests/test02_bridges.py

```diff
@@ -0,0 +1,294 @@
+                                                            containername=C_name))
+            C_client = self.pyclient.get_container_client(C_name)
+            time.sleep(4)
+            if not nat:
+                self.lg.info('Check that C{} can connect to internet ,should fail.'.format(i))
+                response = C_client.bash("ping -c1 8.8.8.8").get()
+                self.assertEqual(response.state, "ERROR", response.stdout)
+            else:
+
+                self.lg.info('Check that C{} can connect to internet ,should succeed.'.format(i))
+                response = C_client.bash("ping -c1 8.8.8.8").get()
+                self.assertEqual(response.state, "SUCCESS", response.stdout)
+
+            self.lg.info('Delete created bridge (B{0}) and container (c{0}), should succeed'.format(i))
+            response = self.bridges_api.delete_nodes_bridges_bridgeid(self.nodeid, B_name)
+            self.assertEqual(response.status_code, 204)
+            response = self.containers_api.delete_containers_containerid(self.nodeid,
+                                                                         C_name)
+            self.assertTrue(response.status_code, 204)
+
+    def test003_create_bridge_with_hwaddr(self):
+        """ GAT-003
+        *Create bridge with hardware address *
+
+        **Test Scenario:**
+
+        #. Create bridge (B0) with specefic hardware address,should succeed.
+        #. Check that bridge created with this hardware address, should succeed.
+        #. Create bridge (B1) with wrong hardware address, should fail.
+
+        """
+        hardwareaddr = self.randomMAC()
+        self.bridge_body["hwaddr"] = hardwareaddr
+        self.lg.info("Create bridge (B0) with specefic hardware address,should succeed.")
+        response = self.bridges_api.post_nodes_bridges(self.nodeid,self.bridge_body)
+        self.assertEqual(response.status_code, 201, response.content)
+        time.sleep(3)
+        self.createdbridges.append({"node": self.nodeid, "name": self.bridge_name})
+
+        self.lg.info(" Check that bridge(B0) created with this hardware address, should succeed.")
+        response = self.nodes_api.get_nodes_nodeid_nics(self.nodeid)
+        self.assertEqual(response.status_code, 200)
+        nic = [x for x in response.json() if x["name"] == self.bridge_name][0]
+        self.assertEqual(nic["hardwareaddr"], hardwareaddr)
+
+        self.lg.info("Create bridge (B1) with wrong hardware address, should fail.")
+        hardwareaddr = self.rand_str()
+        B1_name = self.rand_str()
+        self.bridge_body["hwaddr"] = hardwareaddr
+        self.bridge_body["name"] = B1_name
+        response = self.bridges_api.post_nodes_bridges(self.nodeid, self.bridge_body)
+        self.createdbridges.append({"node": self.nodeid, "name": self.bridge_name})
+        self.assertEqual(response.status_code, 400, response.content)
+
+    def test004_create_bridge_with_static_networkMode(self):
```

Comment:
Divide this tes tcase to 5 test cases cause there is no need to  compress those 5 scenarios in the same test case